### PR TITLE
Added readme for payment service documenting feature flags

### PR DESCRIPTION
**Pull request by maaktweluit**
Based on feedback from the market team we documented our feature flags.

### CODE REVIEW

**Review by Wiezzel — DISMISSED** [review]

File: core/payment/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## DO NOT USE DUMMY DRIVER FOR BUILDS THAT WILL BE DISTRIBUTED!!!
+
+You can enable multiple drivers at the same time, use this table for the required feature flags and platform parameters:
+
+|Driver name|Platform name|Feature flag|Public explorer|Local|Testnet|Mainnet|
+|-|-|-|-|-|-|-|
+|zk-ngnt|ZK-NGNT|`ya-zksync-driver`|[zkscan](https://rinkeby.zkscan.io/)|x|x||
+|ngnt|NGNT|`ya-ngnt-driver`|[etherscan](https://rinkeby.etherscan.io/token/0xd94e3dc39d4cad1dad634e7eb585a57a19dc7efe)|x|x||
+|dummy|DUMMY|`ya-dummy-driver`|None|x|||
```
Comment (tworec) [inline core/payment/README.md:24]: I'd add here a full command line to change the default and select other set of drivers.

**Review by tworec — CHANGES_REQUESTED** [review]

File: core/payment/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Payment Service
+
+This crate is a service to be loaded in Yagna to handle payment scenario's.
+The payment service is the main service `yagna` will be talking to, but not directly handling the payments.
+The payments are made by drivers loaded in the service.
```
[the remark this answers is not in the document]
Comment (Wiezzel) [reply core/payment/README.md:5]: This goes way outside the scope of this PR. It is meant to describe payment driver feature flags.

File: core/payment/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+### Drivers
+
+Currently these drivers are available to use:
+- NGNT
+- Dummy
+- ZK-NGNT
+
+By default there is no driver selected, drivers need to be specifically loaded with a feature flag.
```
[the remark this answers is not in the document]
Comment (Wiezzel) [reply core/payment/README.md:14]: Actually, NGNT driver feature flag is enabled by default.

File: core/payment/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (maaktweluit) [reply core/payment/README.md:5]: Will make an issue on the backlog for this :)

File: core/payment/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (maaktweluit) [reply core/payment/README.md:14]: @Wiezzel  are you sure?
Checking the code it looks as it required the feature:
https://github.com/golemfactory/yagna/blob/master/core/serv/src/main.rs#L212
And its not enabled by default here:
https://github.com/golemfactory/yagna/blob/master/Cargo.toml#L30
Or a i missing something?

@tworec we can change this when making the zsync driver default at the end of the current sprint ( 2 weeks )

File: core/payment/README.md (hunk 1 above)
Comment (maaktweluit) [reply core/payment/README.md:24]: good idea! will do

File: core/payment/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tworec) [reply core/payment/README.md:5]: 👍

File: core/payment/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tworec) [reply core/payment/README.md:14]: @Wiezzel is right. NGNT driver is enabled by default
see
https://github.com/golemfactory/yagna/blob/881188393c2e4d7e7d86d9fbd9e199d10c517043/Cargo.toml#L13
and
https://github.com/golemfactory/yagna/blob/881188393c2e4d7e7d86d9fbd9e199d10c517043/Cargo.toml#L18

[the remark this answers is not in the document]
Comment (maaktweluit) [reply core/payment/README.md:14]: Thanks! missed that line..
Updated the readme accordingly

File: core/payment/README.md (hunk 1 above)
Comment (maaktweluit) [reply core/payment/README.md:24]: updated

File: core/payment/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (maaktweluit) [reply core/payment/README.md:5]: https://golemproject.atlassian.net/browse/PAY-111

**Review by Wiezzel — APPROVED** [review]

**Review by tworec — APPROVED** [review]
lgtm.
sorry for the delay. missed the notification

**Pull request merged**